core/plugins/org.polarsys.capella.core.re/plugin.xml

```diff
@@ -454,11 +454,11 @@
             mapping="org.polarsys.capella.common.re"
             purpose="org.polarsys.capella.common.re.scopeElements">
          <scopeRetriever
-               class="org.polarsys.capella.core.re.handlers.scope.PartTypeRetriever"
+               class="org.polarsys.capella.core.transition.system.handlers.scope.PartTypeRetriever"
                id="org.polarsys.capella.core.re.handlers.scope.PartTypeRetriever">
          </scopeRetriever>
          <scopeRetriever
-               class="org.polarsys.capella.core.re.handlers.scope.DeployedElementRetriever"
+               class="org.polarsys.capella.core.transition.system.handlers.scope.DeployedElementRetriever"
                id="org.polarsys.capella.core.re.handlers.scope.DeployedElementRetriever">

```

Comment:
same here

Reply:
Same, the re plugin declares the scoperetriever in the given mapping, reusing the implementation class